### DISCUSSION TITLE
RangeError: Maximum call stack size exceeded

### DISCUSSION THREAD

**Discussion by devEGBH** [vercel/community#5899] [Help] [answered]
### Summary

![image](https://github.com/vercel/community/assets/122989705/b0ebf11a-1b12-4613-925d-f5e2c1f17667)

https://metashop-sigma.vercel.app/

### Example

https://metashop-sigma.vercel.app/

### Steps to Reproduce

rebuild again

**Comment by swarnava** [comment, marked as answer]
Hi,

This issue has been fixed in Next.js v14.1. Please give it a try and let us know how it goes.

**Reply by devEGBH** [reply]
![image](https://github.com/vercel/community/assets/122989705/b24288a0-7615-4dfd-a0c0-7cdf5a70e691)
it is showing error,  i am using "next": "14.0.4",

**Reply by swarnava** [reply]
You need to upgrade to v `14.1`. v `14.0.4`  is affected and still cause the error :)

**Reply by devEGBH** [reply]
![image](https://github.com/vercel/community/assets/122989705/f6a31a2d-28c5-4d80-973b-248764d0ec21)

![image](https://github.com/vercel/community/assets/122989705/dced0df1-865b-4255-865b-902a4edd4cc0)

inner apps:
![image](https://github.com/vercel/community/assets/122989705/e16818b4-40e6-43f7-b838-b39451ca2c41)

![image](https://github.com/vercel/community/assets/122989705/1995e44e-f7c3-4a50-9204-51004330fdc1)

what path should we add

it should be :  apps/site/public is it correct?

**Reply by swarnava** [reply]
It should be in your package.json :) You can also run `pnpm up next react react-dom eslint-config-next --latest`

https://nextjs.org/docs/app/building-your-application/upgrading/version-14

**Reply by swarnava** [reply]
Okay, it looks like you are talking about a different error. You can open a new thread.

**Reply by devEGBH** [reply]
and also it always show this error:
![image](https://github.com/vercel/community/assets/122989705/2424bb38-30ff-4d5d-b929-711b52989c6a)
in vercel after build.  now build is showing green button and say ready but still this error showing can we check what is the error is there any way where I can check what is showing 404
